Makefile.old

```diff
@@ -62,17 +65,19 @@ clean-db: services-clean-db
 clean-all: clean clean-db
 
 check: test
-test: build ui-test services-test pp-test plugins-test
+test: build services-test pp-test plugins-test
 	bin/HootTest $(HOOT_TEST_DIFF) --slow
 
 docs: always HOOT_VERSION_FILE services-docs
+	echo Building documentation...
 	cd docs; $(MAKE)
 
-test-quick: build services-test pp-test
+test-quick: build pp-test
 	bin/HootTest $(HOOT_TEST_DIFF) --quick
 
-test-all-no-core: ui-test build services-test-all pp-test plugins-test
-
+test-all-no-core: services-test-all pp-test plugins-test
+	make ui-test
```

Comment:
You should be able to use a | to make things sequential. E.g.

```
test-all-no-core: serivces-test-all pp-test plugins-test | ui-test
```

I think the above will run ui-test only after all the other jobs are complete. Though we're getting into some of the esoteric Makefile options that I don't use too often. It may need to be like this:

```
test-all-no-core: pp-test plugins-test | serivces-test-all ui-test
```